README.md

```diff
@@ -5,12 +5,7 @@
   </p>
   <br />
   <p>
-    <a href="https://discord.gg/bRCvFy9"><img src="https://discordapp.com/api/guilds/222078108977594368/embed.png" alt="Discord server" /></a>
-    <a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/v/discord.js.svg?maxAge=3600" alt="NPM version" /></a>
-    <a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/dt/discord.js.svg?maxAge=3600" alt="NPM downloads" /></a>
-    <a href="https://travis-ci.org/discordjs/discord.js"><img src="https://travis-ci.org/discordjs/discord.js.svg" alt="Build status" /></a>
-    <a href="https://david-dm.org/discordjs/discord.js"><img src="https://img.shields.io/david/discordjs/discord.js.svg?maxAge=3600" alt="Dependencies" /></a>
-    <a href="https://www.patreon.com/discordjs"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg" alt="Patreon" /></a>
+    <a href="https://discord.gg/bRCvFy9"><img src="https://img.shields.io/discord/222078108977594368.svg?style=flat-square" alt="Discord server" /></a><a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/v/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM version" /></a><a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/dt/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM downloads" /></a><a href="https://travis-ci.org/discordjs/discord.js"><img src="https://img.shields.io/travis/discordjs/discord.js.svg?style=flat-square" alt="Build status" /></a><a href="https://david-dm.org/discordjs/discord.js"><img src="https://img.shields.io/david/discordjs/discord.js.svg?style=flat-square" alt="Dependencies" /></a><a href="https://www.patreon.com/discordjs"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg?style=flat-square" alt="Patreon" /></a>
```

Comment:
You should probably put all this HTML on separate lines rather than having it all bunched up.

Reply:
Sadly that would undermine the whole goal of this PR, to make the badges look nice. Spacing them out gives this as result:

```html
    <div align="center">
        <p>
            <a href="https://discord.gg/bRCvFy9"><img src="https://img.shields.io/discord/222078108977594368.svg?style=flat-square" alt="Discord server" /></a>
            <a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/v/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM version" /></a>
            <a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/dt/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM downloads" /></a>
            <a href="https://travis-ci.org/discordjs/discord.js"><img src="https://img.shields.io/travis/discordjs/discord.js.svg?style=flat-square" alt="Build status" /></a>
            <a href="https://david-dm.org/discordjs/discord.js"><img src="https://img.shields.io/david/discordjs/discord.js.svg?style=flat-square" alt="Dependencies" /></a>
            <a href="https://www.patreon.com/discordjs"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg?style=flat-square" alt="Patreon" /></a>
        </p>
    </div>
```


<div align="center">
<p>
<a href="https://discord.gg/bRCvFy9"><img src="https://img.shields.io/discord/222078108977594368.svg?style=flat-square" alt="Discord server" /></a>
<a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/v/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM version" /></a>
<a href="https://www.npmjs.com/package/discord.js"><img src="https://img.shields.io/npm/dt/discord.js.svg?style=flat-square&maxAge=3600" alt="NPM downloads" /></a>
<a href="https://travis-ci.org/discordjs/discord.js"><img src="https://img.shields.io/travis/discordjs/discord.js.svg?style=flat-square" alt="Build status" /></a>
<a href="https://david-dm.org/discordjs/discord.js"><img src="https://img.shields.io/david/discordjs/discord.js.svg?style=flat-square" alt="Dependencies" /></a>
<a href="https://www.patreon.com/discordjs"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg?style=flat-square" alt="Patreon" /></a>
</p>
</div>

And since [github does not support](https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/sanitization_filter.rb#L44-L106) white-space pre-wrap and word-wrap none [styling](https://stackoverflow.com/questions/11599859/forcing-a-long-line-of-text-without-spaces-to-line-break-according-to-parent-c) the only way is to 1 line it.

Reply:
You could comment out the whitespace between each badge.

Reply:
they look way better spaced out, but still worse than the current badges